uobtheatre/bookings/templates/emails/booking_confirmation_email.txt

```diff
@@ -0,0 +1,6 @@
+Hello {{ user_name }}!
+
```

Comment:
Surely we can get the plain text content from just stripping the HTML tags? Seems very weird to have to write this twice

Reply:
This assumes the HTML and text versions are the same. I hope someday when we write a proper email the HTML template will deviate maybe with some nice images etc